bin/testObservability/helper/helper.js

```diff
@@ -351,9 +351,43 @@ const getCiInfo = () => {
 
 let packages = {};
 
-exports.getPackageVersion = (package_) => {
+exports.getPackageVersion = (package_, bsConfig = null) => {
   if(packages[package_]) return packages[package_];
-  return packages[package_] = this.requireModule(`${package_}/package.json`).version;
+  let packageVersion;
+  /* Try to find version from module path */
+  try {
+    packages[package_] = this.requireModule(`${package_}/package.json`).version;
+    logger.info(`Getting ${package_} package version from module path = ${packages[package_]}`);
+    packageVersion = packages[package_];
+  } catch(e) {
+    exports.debug(`Unable to find package ${package_} at module path with error ${e}`);
```

Comment:
wont it just print [OBJECT][OBJECT] and not the error message ?

Reply:
Nope it prints the first line of the error stacktrace.